server_test.go

```diff
@@ -575,6 +586,98 @@ func TestShutdownTCP(t *testing.T) {
 	}
 }
 
+func testLostQueriesAtShutdownServer(t *testing.T, srv *Server, addr string, fin chan error, client *Client) {
```

Comment:
this is not tested because test is lowercase

Reply:
yes. it is a common function that will be called by 3 test cases (UDP. TCP. TLS)